collection_flow_layout/Classes/PinterestStyleFlowLayout/PinterestStyleFlowLayout.swift

```diff
@@ -42,27 +42,23 @@ public class PinterestStyleFlowLayout: ContentDynamicLayout {
 
             attributes.frame.origin.x = CGFloat(currentColumnIndex) * (cellWidth + cellsPadding.horizontal) + contentPadding.horizontal
 
-            previousCellsYOffset[currentColumnIndex] = (cellSize?.height ?? 0) + previousCellsYOffset[currentColumnIndex] + cellsPadding.vertical
+            previousCellsYOffset[currentColumnIndex] = cellSize.height + previousCellsYOffset[currentColumnIndex] + cellsPadding.vertical
 
             addCachedLayoutAttributes(attributes: attributes)
         }
 
-        contentSize.height = (previousCellsYOffset.max() ?? 0) + contentPadding.vertical - cellsPadding.vertical
+        contentSize.height = previousCellsYOffset.max()! + contentPadding.vertical - cellsPadding.vertical
     }
 
     private func minYOffsetFrom(array: [CGFloat]) -> (offset: CGFloat, index: Int) {
-        let minYOffset = array.min() ?? 0
-        let minIndex = array.index(of: minYOffset) ?? 0
+        let minYOffset = array.min()!
```

Comment:
This code will crash with empty array

Reply:
@plzen No, because this method is called inside for-in loop and only in case we have at least one item.

Reply:
OK